Sources/ConsoleLogger/ConsoleLogger.swift

```diff
@@ -14,25 +13,23 @@ public struct ConsoleLogger<T: LoggerFragment>: LogHandler, Sendable {
     /// See `LogHandler.logLevel`.
     public var logLevel: Logger.Level
 
-    /// The conosle that the messages will get logged to.
-    public let console: any Console
-
     /// The ``LoggerFragment`` this logger outputs through.
     public var fragment: T
 
+    /// The printer used to output log messages. Used for testing purposes only.
+    package var printer: any ConsoleLoggerPrinter = DefaultConsoleLoggerPrinter()
```

Comment:
Let's make this private but have it passed into the initialiser so that anyone can provide the out printer if they want